app/services/course/assessment/answer/text_response_comprehension_auto_grading_service.rb

```diff
@@ -231,10 +233,10 @@ def explanations_for_keyword(answer_text_array, status)
         next if status[index].nil?
 
         word_explanation = answer_text_array[index]
-        word_explanation += " (#{status[index].explanation})" unless status[index].explanation.nil?
+        # word_explanation += " (#{status[index].explanation})" unless status[index].explanation.nil?
```

Comment:
Is this temporary? Could you put a comment before this line to explain why it is commented out instead of removed?